src/monitor.cc

```diff
@@ -485,16 +553,27 @@ bool NodeMonitor::getBooleanFunction(const char* funcName) {
     return false;
 }
 
+NodeMonitor& NodeMonitor::getInstance() {
+    assert( 0 != instance_ );
+    return *instance_;
+}
+
+/**
+ * Sends udp JSON datagram approximately once per REPORT_INTERVAL_MS
+ *
+ * Executed in the monitr pthread, *not* from any pthread executing v8
+ * Therefore, we can't directly access any Javascript functions/vars
+ **/
 bool NodeMonitor::sendReport() {
     static pid_t pid = getpid();
     static double minOverHead = 0;
 
-    // See how many reports has been processed since last call to this function
-    unsigned int diff_count = instance_->loop_count_ - instance_->last_loop_count_ + 1;
-    instance_->last_loop_count_ = instance_->loop_count_;
+    // See how many reports have been processed since last call to this function
+    unsigned int diff_count = loop_count_ - last_loop_count_ + 1;
+    last_loop_count_ = loop_count_;
 
     // The different between current and previous time (in miliseconds)
```

Comment:
`start_timestamp_` is set to `uv_hrtime()` each time a report is sent.  `loop_timestamp_` is set to `uv_hrtime()` each time through the loop.

`uv_hrtime()` is defined to return # of *nanoseconds*.  See http://docs.libuv.org/en/latest/misc.html

So dividing by 1.0e6 is equivalent to returning milliseconds.  (10^9 / 10^6 = 10^3)